rspec-puppet-facts.gemspec

```diff
@@ -28,5 +28,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'deep_merge', '~> 1.2'
   s.add_dependency 'facter', '< 5'
   s.add_dependency 'facterdb', '~> 3.1'
-  s.add_dependency 'puppet', '>= 7', '< 9'
+  s.add_dependency 'openvox', '~> 8.19'
```

Comment:
I thought about dropping the dependency altogether and require it at runtime with a fallback. While doing so we should also report the implementation as a fact

Reply:
https://github.com/voxpupuli/rspec-puppet-facts/pull/213